<launch>
	<arg name="model" />
	<param name="robot_description" command="$(find xacro)/xacro.py $(find shuk_face)/urdf/shuk_face.xacro" /> 
	<param name="use_gui" value="true"/>
	<node name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher" />
	<node name="robot_state_publisher" pkg="robot_state_publisher" type="state_publisher" />
	<node name="rviz" pkg="rviz" type="rviz" args="-d $(find shuk_face)/face.rviz" required="true" />
</launch>
